spec/features/single_content_item_spec.rb

```diff
@@ -251,6 +251,30 @@
       it 'renders the publishing application' do
         expect(page).to have_selector '.related-actions', text: 'Whitehall'
       end
+
+      it 'renders the contacts application' do
+        stub_metrics_page(base_path: 'contacts/path', time_period: :past_30_days, publishing_app: 'contacts')
+        visit '/metrics/contacts/path'
+        expect(page).to have_link("Edit in Contacts", href: 'http://contacts-admin.dev.gov.uk/admin/contacts/path/edit')
+      end
+
+      it 'renders the specialist publisher application' do
+        stub_metrics_page(base_path: 'specialist/path', time_period: :past_30_days, publishing_app: 'specialist-publisher')
+        visit '/metrics/specialist/path'
+        expect(page).to have_link("Edit in Specialist publisher", href: "http://specialist-publisher.dev.gov.uk/news-storys/content-id/edit")
+      end
+
+      it 'renders the collections application' do
+        stub_metrics_page(base_path: 'collections/path', time_period: :past_30_days, publishing_app: 'collections-publisher')
+        visit '/metrics/collections/path'
+        expect(page).to have_link("Edit in Collections publisher", href: 'http://support.dev.gov.uk/general_request/new')
+      end
+
+      it 'renders the travel advice application' do
+        stub_metrics_page(base_path: 'travel/path', time_period: :past_30_days, publishing_app: 'travel-advice-publisher')
+        visit '/metrics/travel/path'
+        expect(page).to have_link("Edit in Travel advice publisher", href: 'http://travel-advice-publisher.dev.gov.uk/admin/path')
+      end
```

Comment:
I'm unsure about the value of having these tests here, as they seem to duplicate some of the tests in the `external_links_helper_spec.rb` file. Maybe just test one case here?

Reply:
I think there is some value in showing the combination of the link to publishing applications appearing on the pages we expect it to, with the correct label, and the correct link. 

IMO it does only add a little value though so if you feel that the extra time it takes for the tests to run, or other potential downswides to having them in, outweigh that minor benefit then I am happy to remove them!

Reply:
> I mind keeping these particular tests.

That's meant to be: I _don't_ mind keeping these particular tests.